lerna.json

```diff
@@ -1,8 +1,22 @@
 {
   "version": "2.5.10-alpha.0",
   "npmClient": "yarn",
+  "packages": [
+    "packages/backend-core",
```

Comment:
I assume we can change this back once we get account-portal/pro into a standard shape?

Reply:
This is because of the shape of the pro only. Without this will treat as a package the path in `pacakge/pro`, that is not a valid package. Once this out we can refactor pro repo to be a single package, changing this back to `packages/*`